docs/cross-contract/callbacks.md

```diff
@@ -4,10 +4,48 @@ sidebar_position: 2
 
 # Callbacks
 
-<!-- What are callbacks -->
+NEAR Protocol is a sharded, proof-of-stake blockchain that behaves differently than proof-of-work blockchains. When interacting with a native Rust (compiled to Wasm) smart contract, cross-contract calls are asynchronous. Callbacks are used to either get the result of a cross-contract call or tell if a cross-contract call has succeeded or failed.
```

Comment:
I mean, the PoW vs PoS doesn't affect the need for asynchronous calls, does it? If anything, a sharded PoW chain would gain more value from async calls, I think.